unit-tests/readme.md

```diff
@@ -11,12 +11,26 @@ Make sure you have Intel® RealSense™ device connected.
 
 If not all unit-tests are passing this can be related to faulty device or problems with the environment setup. 
 We support recording test flows at the backend level and running tests on top of mock hardware. This helps us distingwish between hardware problems and software issues. 
-You can record unit-test into file using:
+
+* You can record unit-test into file using:
 `live-test into filename`
-To run unit-tests without actual hardware, based on recorded data, run:
+
+* To run unit-tests without actual hardware, based on recorded data, run:
 `live-test from filename`
```

Comment:
I think it would be clearer to write:
`./live-test from <filename>`
(notice the `./` and the angle brackets)